wiz/table_wiz_security_framework.go

```diff
@@ -0,0 +1,134 @@
+package wiz
+
+import (
+	"context"
+
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-wiz/api"
+)
+
+//// TABLE DEFINITION
+
+func tableWizSecurityFramework(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "wiz_security_framework",
+		Description: "Wiz Security Framework",
+		List: &plugin.ListConfig{
+			Hydrate: listWizSecurityFrameworks,
+			KeyColumns: plugin.KeyColumnSlice{
+				{Name: "enabled", Require: plugin.Optional, Operators: []string{"<>", "="}},
+			},
+		},
+		Get: &plugin.GetConfig{
+			Hydrate:    getSecurityFramework,
+			KeyColumns: plugin.SingleColumn("id"),
+		},
+		Columns: []*plugin.Column{
+			{Name: "name", Type: proto.ColumnType_STRING, Description: "The name of the security framework."},
+			{Name: "id", Type: proto.ColumnType_STRING, Description: "A unique identifier of the security framework."},
+			{Name: "enabled", Type: proto.ColumnType_BOOL, Description: "If true, the security framework is enabled."},
+			{Name: "built_in", Type: proto.ColumnType_BOOL, Description: "If true, the security framework is managed by Wiz."},
+			{Name: "description", Type: proto.ColumnType_STRING, Description: "The description of the security framework."},
+			{Name: "policy_types", Type: proto.ColumnType_JSON, Description: "A list of security framework policy type."},
```

Comment:
Does the `policy_types` column need no transformation? Can we query its components like any other IAM policy?

Reply:
@misraved the policy types returns an array of policy types that indicates the type of resource to which a security policy applies.